rabbit-escape-render/src/rabbitescape/render/gameloop/WaterDynamics.java

```diff
@@ -0,0 +1,157 @@
+package rabbitescape.render.gameloop;
+
+import java.util.List;
+import java.util.Vector;
+
+import static rabbitescape.engine.CellularDirection.*;
+
+import rabbitescape.engine.WaterRegion;
+import rabbitescape.engine.World;
+import rabbitescape.engine.util.CellDebugPrint;
+import rabbitescape.engine.util.Dimension;
+import rabbitescape.engine.util.LookupTable2D;
+import rabbitescape.render.Polygon;
+import rabbitescape.render.WaterRegionRenderer;
+
+/**
+ * Manages animation of polygons for water.
+ */
+public class WaterDynamics
```

Comment:
Maybe this should be called `WaterAnimation`?  I started thinking it controlled the movement of water, not how it looks.